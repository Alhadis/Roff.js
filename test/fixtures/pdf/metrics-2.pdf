%PDF-1.4
%‚„œ”
3 0 obj << /Contents [4 0 R  ]
/Group << /CS /DeviceRGB
/S /Transparency
>>
/Parent 2 0 R
/Type /Page
>>
endobj
4 0 obj << /Length 1975
>>
stream
q 1 0 0 1 0 0 cm
1 J
1 j
% V12000
%% H72000
% DFd
0 g
% H1440
% V-52800
%% md
% md
0 G
% DFr 48830 0 0
0.745098039215686 0 0 rg
% s10000
% DP 0 820800 608400 0 0 -820800 -608400 0
1.44 844.8 m
1.44 24 l
609.84 24 l
609.84 844.8 l
1.44000000000005 844.8 l
f
% n12000 0
q BT
1 0 0 1 1.44000000000005 844.8 Tm
0 g
0 Tc
% x font 5 TR
% f5
% s72000
/F5 10 Tf
% V84000
/F5 72 Tf
%% H72000
% tP
1 0 0 1 72 708 Tm
0 Tc
% == 'P'=40.032 'xpos=112.032
% --- wht=18000, pend=0, nomv=0
% H105408
0 Tw (P) Tj
33.408 0 Td
% tA
% == 'A'=51.984 'xpos=157.392
% --- wht=18000, pend=0, nomv=0
% H154512
0 Tw (A) Tj
49.104 0 Td
% tGE
% == 'GE'=95.976 'xpos=250.488
% --- wht=18000, pend=0, nomv=0
% h18000
% tWIDTH
% == 'WIDTH'=239.904 'xpos=508.392
% --- wht=18000, pend=18000, nomv=0
% h18000
% t=
% == '='=40.608 'xpos=567
% --- wht=18000, pend=18000, nomv=0
% n72000 0
0 Tw [(GE) -250 (WIDTH) -250 (=) ] TJ
% V156000
%% H72000
% t8.5
1 0 0 1 72 636 Tm
0 Tc
% == '8.5'=90 'xpos=162
% --- wht=18000, pend=0, nomv=0
% h18000
% tinches
% == 'inches'=183.96 'xpos=363.96
% --- wht=18000, pend=18000, nomv=0
% n72000 0
0 Tw [(8.5) -250 (inches) ] TJ
% V372000
%% H72000
% tP
1 0 0 1 72 420 Tm
0 Tc
% == 'P'=40.032 'xpos=112.032
% --- wht=18000, pend=0, nomv=0
% H105408
0 Tw (P) Tj
33.408 0 Td
% C[
% == '['=23.976 'xpos=129.384
% --- wht=18000, pend=0, nomv=0
% H154512
0 Tw [([) 333 ] TJ
49.104 0 Td
% tGE
% == 'GE'=95.976 'xpos=250.488
% --- wht=18000, pend=0, nomv=0
% h18000
% tHEIGHT
% == 'HEIGHT'=267.912 'xpos=536.4
% --- wht=18000, pend=18000, nomv=0
% h18000
% t=
% == '='=40.608 'xpos=595.008
% --- wht=18000, pend=18000, nomv=0
% n72000 0
0 Tw [(GE) -250 (HEIGHT) -250 (=) ] TJ
% V444000
%% H72000
% t11
1 0 0 1 72 348 Tm
0 Tc
% == '11'=72 'xpos=144
% --- wht=18000, pend=0, nomv=0
% h18000
% tinches
% == 'inches'=183.96 'xpos=345.96
% --- wht=18000, pend=18000, nomv=0
% n72000 0
0 Tw [(11) -250 (inches) ] TJ
% x trailer
% V792000
%% x stop
% x stop
1 0 0 1 345.96 0 Tm
0 Tc
ET Q
Q
endstream
endobj
1 0 obj << /Pages 2 0 R 
/Type /Catalog
>>
endobj
8 0 obj << /CreationDate (D:20170719171227+10'00')
/Creator (groff version 1.22.3)
/ModDate (D:20170719171227+10'00')
/Producer (gropdf version 1.22.3)
>>
endobj
5 0 obj << /BaseFont /Times-Roman
/Encoding 6 0 R 
/Subtype /Type1
/ToUnicode 7 0 R 
/Type /Font
>>
endobj
6 0 obj << /Differences [0 /asciicircum /asciitilde /Scaron /Zcaron /scaron /zcaron /Ydieresis /trademark /quotesingle /Euro /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /.notdef /space /exclam /quotedbl /numbersign /dollar /percent /ampersand /quoteright /parenleft /parenright /asterisk /plus /comma /hyphen /period /slash /zero /one /two /three /four /five /six /seven /eight /nine /colon /semicolon /less /equal /greater /question /at /A /B /C /D /E /F /G /H /I /J /K /L /M /N /O /P /Q /R /S /T /U /V /W /X /Y /Z /bracketleft /backslash /bracketright /circumflex /underscore /quoteleft /a /b /c /d /e /f /g /h /i /j /k /l /m /n /o /p /q /r /s /t /u /v /w /x /y /z /braceleft /bar /braceright /tilde /.notdef /quotesinglbase /guillemotleft /guillemotright /bullet /florin /fraction /perthousand /dagger /daggerdbl /endash /emdash /.notdef /fi /fl /.notdef /.notdef /dotlessi /.notdef /grave /hungarumlaut /dotaccent /breve /caron /ring /ogonek /quotedblleft /quotedblright /oe /lslash /quotedblbase /OE /Lslash /.notdef /exclamdown /cent /sterling /currency /yen /brokenbar /section /dieresis /copyright /ordfeminine /guilsinglleft /logicalnot /minus /registered /macron /degree /plusminus /twosuperior /threesuperior /acute /mu /paragraph /periodcentered /cedilla /onesuperior /ordmasculine /guilsinglright /onequarter /onehalf /threequarters /questiondown /Agrave /Aacute /Acircumflex /Atilde /Adieresis /Aring /AE /Ccedilla /Egrave /Eacute /Ecircumflex /Edieresis /Igrave /Iacute /Icircumflex /Idieresis /Eth /Ntilde /Ograve /Oacute /Ocircumflex /Otilde /Odieresis /multiply /Oslash /Ugrave /Uacute /Ucircumflex /Udieresis /Yacute /Thorn /germandbls /agrave /aacute /acircumflex /atilde /adieresis /aring /ae /ccedilla /egrave /eacute /ecircumflex /edieresis /igrave /iacute /icircumflex /idieresis /eth /ntilde /ograve /oacute /ocircumflex /otilde /odieresis /divide /oslash /ugrave /uacute /ucircumflex /udieresis /yacute /thorn /ydieresis /ellipsis ]
/Type /Encoding
>>
endobj
7 0 obj << /Length 426
>>
stream
/CIDInit /ProcSet findresource begin
12 dict begin
begincmap
/CIDSystemInfo
<< /Registry (Adobe)
/Ordering (UCS)
/Supplement 0
>> def
/CMapName /Adobe-Identity-UCS def
/CMapType 2 def
1 begincodespacerange
<0000> <FFFF>
endcodespacerange
2 beginbfrange
<008b> <008f> [<00660066> <00660069> <0066006c> <006600660069> <00660066006C>]
<00ad> <00ad> <002d>
endbfrange
endcmap
CMapName currentdict /CMap defineresource pop
end
end
endstream
endobj
2 0 obj << /Count 1
/Kids [3 0 R  ]
/MediaBox [0 0 612 792 ]
/Resources << /Font << /F5 5 0 R 
>>
/ProcSet [/PDF /Text /ImageB /ImageC /ImageI ]
>>
/Rotate 0
/Type /Pages
>>
endobj
xref
0 9
0000000000 65535 f 
0000002153 00000 n 
0000005078 00000 n 
0000000015 00000 n 
0000000127 00000 n 
0000002365 00000 n 
0000002472 00000 n 
0000004602 00000 n 
0000002203 00000 n 
trailer
<<
/Info 8 0 R 
/Root 1 0 R
/Size 9
>>
startxref
5259
%%EOF
