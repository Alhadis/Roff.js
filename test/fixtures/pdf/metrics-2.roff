.nf
\v'-0.9i'\
\h'-0.98i'\
\M[grey]\
\D'P0 11.4i 8.45i 0 0 -11.4i -8.45i 0'
.ps 72p
.vs 1m
PAGE\~WIDTH\ =
.in
.br
8.5\~inches
.sp 2
PAGE\~HEIGHT\ =
.br
11\~inches
