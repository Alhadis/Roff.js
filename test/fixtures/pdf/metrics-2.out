x T pdf
x res 72000 1 1
x init
p1
V12000
H72000
DFd
H1440
V-52800
md
DFr 48830 0 0
s10000
DP 0 820800 608400 0 0 -820800 -608400 0
n12000 0
x font 5 TR
f5
s72000
V84000
H72000
tP
H105408
tA
H154512
tGE
h18000
tWIDTH
h18000
t=
n72000 0
V156000
H72000
t8.5
h18000
tinches
n72000 0
V372000
H72000
tP
H105408
C[
H154512
tGE
h18000
tHEIGHT
h18000
t=
n72000 0
V444000
H72000
t11
h18000
tinches
n72000 0
x trailer
V792000
x stop
